### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by jalarg**
# Summary
- Add footer component
- Add github and Copyright icon
- Add tests for the different functionalities

# Details
- I added a new footer component to the project. This footer will serve as a valuable addition to the user interface, enhancing navigation and providing users with essential information and links.

# Evidence 

## Stellar wallet: 

Wallet UI

testing

### CODE REVIEW

File: cypress/e2e/home.cy.ts [hunk 1]
```diff
@@ -77,3 +83,42 @@ describe(" Navbar: UI and functionality:", () => {
     cy.get("@themeButtonLight").should("not.exist");
   });
 });
+describe(" Footer: UI and functionality:", () => {
+  it("Should the footer container be visible", () => {
+    cy.get(".footer-container").should("exist").and("be.visible");
+  });
+
+  it("Should the github logo be visible and have a href", () => {
```
Comment (Genebson) [inline cypress/e2e/home.cy.ts]: `...and have the correct url`?

File: cypress/e2e/home.cy.ts [hunk 2]
```diff
@@ -77,3 +83,42 @@ describe(" Navbar: UI and functionality:", () => {
...
+    cy.get(".github-icon")
+      .should("be.visible")
+      .and("have.attr", "href", githubUrl);
+  });
+
+  it("Should the terms and services and have a href", () => {
```
Comment (Genebson) [inline cypress/e2e/home.cy.ts]: Same here and the ones from below.

File: src/pages/index.tsx [hunk 3]
```diff
@@ -6,87 +6,89 @@ import Button from "../components/commons/Button";
...
+                  >
+                    Confirm wallet
+                  </Button>
+                </div>
+                <div className="flex flex-col justify-center m-5">
+                  <Button
+                    buttonClass="button-go-back"
+                    onClick={() => {
+                      setPublicKey(null);
+                      setSecret(null);
```
Comment (Genebson) [inline src/pages/index.tsx:68]: We can save this in a function.

**Review by Genebson — APPROVED** [review]

File: cypress/e2e/home.cy.ts (hunk 1 above)
Comment (jalarg) [reply cypress/e2e/home.cy.ts]: Fix in commit e46bc72621e2e6b5b3167b81e5de57d91c8b23d3.

File: src/pages/index.tsx (hunk 3 above)
Comment (jalarg) [reply src/pages/index.tsx:68]: Will be fix in branch feature/modals.

File: cypress/e2e/home.cy.ts (hunk 2 above)
Comment (jalarg) [reply cypress/e2e/home.cy.ts]: Fix in commit e46bc72621e2e6b5b3167b81e5de57d91c8b23d3.

**Pull request merged**